frontend/pages/club/[club]/application/[application]/index.tsx

```diff
@@ -194,24 +194,6 @@ const ApplicationPage = ({
               }
             }
 
-            // submissions open & close error check
```

Comment:
Why are we getting rid of this?

Reply:
Wasn't sure why this was necessary since we are performing validation on the backend, and adding the extension logic requires changing this check on the frontend

Reply:
I think I initially added on the frontend because it would save us an unnecessary call to the backend. But if the frontend is no longer doing bookkeeping on extensions, then I'm ok with this